projectgenerator/libili2pg/ilicache.py

```diff
@@ -130,29 +132,48 @@ def process_local_ili_folder(self, path):
         '''
         Parses all .ili files in the given ``path`` (non-recursively)
         '''
-        model = dict()
         models = list()
-        re_model = re.compile('\s*MODEL\s*([\w\d_-]+)\s.*')
-        re_model_version = re.compile('VERSION "([ \w\d\._-]+)".*')
-
+        fileModels = list()
         for ilifile in glob.iglob(os.path.join(path, '*.ili')):
-            with open(ilifile, 'r') as file:
-                for line in file:
-                    result = re_model.search(line)
-                    if result:
-                        model = dict()
-                        model['name'] = result.group(1)
-                        model['version'] = ''
-                        models += [model]
-
-                    result = re_model_version.search(line)
-                    if result:
-                        model['version'] = result.group(1)
-
-        print(repr(models[0]['name']))
+            try:
+                fileModels = self.parse_ili_file(ilifile, "utf-8")
+            except UnicodeDecodeError:
+                try:
+                    fileModels = self.parse_ili_file(ilifile, "latin1")
+                    self.iface.messageBar().pushWarning(self.tr('Project Generator'),
```

Comment:
Chances are users won't read the message in the log window, as there are too many lines to read already. However, we (Sergio and me) think that the orange color of a `QgsMessageBar` warning is a bit aggressive for the user and might convey the message that the plugin threw errors. 

I'd be in favor of putting it in the log window. If it's not too visible, we may play with another font color.